src/bors/handlers/help.rs

```diff
@@ -0,0 +1,34 @@
+use crate::bors::RepositoryClient;
+use crate::bors::RepositoryState;
+use crate::github::PullRequest;
+
+const HELP_MESSAGE: &str = r#"
+- try: Request that the PR be tested by buildbot, without accepting it.
```

Comment:
```suggestion
- try: Execute the `try` CI workflow on this PR (without approving it for a merge).
```

`buildbot` is actually the name of an existing CI service, which we do not use. This could be a bit confusing to users. We also can't just say to "test this PR on CI", because all PRs are tested on CI, just with different workflows xD So I suggest rewording a bit.

Reply:
Really sorry about that, I copy it from the old homu page without double checking it